### DISCUSSION TITLE
Double Float Percision Completed

### DISCUSSION THREAD

**Discussion by Valinakova** [Hoimar/Planet-Generator#33] [General]
Some good news.

https://github.com/godotengine/godot-proposals/issues/892

**Comment by Hoimar** [comment]
This is indeed good news. Now all that is missing is allowing a scene in a 64bit doubles world to be rendered down to a 32bit representation for the GPU IIRC. This may be possible to do in GDScript using familiar techniques like origin shifting and dynamic camera clipping to improve depth buffer usage.

I might give this another go using the current state of double support. The previous attempt with the then half-baked support wasn't quite successful yet.

**Reply by Valinakova** [reply]
On rendering, I believe it is being discussed here https://github.com/godotengine/godot/issues/58516
